store/src/java/com/zimbra/cs/util/AccountUtil.java

```diff
@@ -1042,4 +1044,19 @@ public static void isAdminAccount(Account acct) throws ServiceException {
         if (!isAdminAccount)
             throw ServiceException.PERM_DENIED("not an admin account");
     }
+    
+    /**
+     * Utils method to translate system folder name
+     * @param name
+     * @param locale
+     * @return
+     */
+    public static String getTranslatedFolderName(String name, Locale locale) {
+        if ("Calendar".equals(name)) {
```

Comment:
can we check on folder id instead of name?

Reply:
In 'name' variable we get 'Calendar' as value. So id can not used here.

Reply:
Can we pass sid when calling this utility, extract the id and check if it matches any of the system folder ids?